src/engraving/dom/note.cpp

```diff
@@ -1405,6 +1405,70 @@ void Note::updateFrettingForTiesAndBends()
     setFret(prevNote->fret());
 }
 
+bool Note::hideFret() const
```

Comment:
For me based on the name of the method, it looks like this method hides fret and does not return an indication that fret needs to be hidden
The same for `forceShowFret`

Reply:
Also good point, thanks 👍